### ISSUE TITLE
yolov3-tiny 检测不出内容

### ISSUE THREAD

**Issue by dd110hh110** [ChenYingpeng/caffe-yolov3#36]
通过转换工具，我只获得了一个tiny-yolo-voc.prototxt 和 tiny-yolo-voc.caffemodel。通过修改detectnet.cpp（减少了一个blob），yolo_layer.cpp（改变循环参数），能够进行检测，但是没有检测到任何东西，请问这是怎么回事呢？

**Comment by passion3394** [comment]
遇到同样的问题，作者能出来解释一下嘛，多谢！

**Comment by ChenYingpeng** [comment]
Please check note 4 Yolov3-tiny: Caffe can not duplicate the layer that maxpool layer (params:kernel_size = 2,stride = 1),so rewrite max_pool_1d function for recurrenting it.
I convert yolov3-tiny.prototxt to yolov3-tiny-1.prototxt and yolov3-tiny-2.prototxt.Detail check Example 4:yolov3-tiny.

**Comment by passion3394** [comment]
yes, I followed the note 4 to split the original yolov3-tiny prototxt to two prototxt, the result is very bad.

**Comment by passion3394** [comment]
the original Darknet model works normally, but the converted yolov3-tiny-1.prototxt and yolov3-tiny-2.prototxt can only detect a point, which is  the center of the object.

**Comment by dd110hh110** [comment]
> 遇到同样的问题，作者能出来解释一下嘛，多谢！



> yes, I followed the note 4 to split the original yolov3-tiny prototxt to two prototxt, the result is very bad.

How do you split the original yolov3-tiny prototxt to two prototxt? I don't understand the specific method very well.

**Comment by dd110hh110** [comment]
> Please check note 4 Yolov3-tiny: Caffe can not duplicate the layer that maxpool layer (params:kernel_size = 2,stride = 1),so rewrite max_pool_1d function for recurrenting it.
> I convert yolov3-tiny.prototxt to yolov3-tiny-1.prototxt and yolov3-tiny-2.prototxt.Detail check Example 4:yolov3-tiny.

I am not very clear how to split the original yolov3-tiny prototxt to two prototxt？Can you explain it briefly?

**Comment by passion3394** [comment]
@dd110hh110 please refer the link on the readme of this repo, the author has published the two spited prototxt in the Baidu net disk.

https://pan.baidu.com/s/1yiCrnmsOm0hbweJBiiUScQ

**Comment by dd110hh110** [comment]
> @dd110hh110 please refer the link on the readme of this repo, the author has published the two spited prototxt in the Baidu net disk.
> 
> https://pan.baidu.com/s/1yiCrnmsOm0hbweJBiiUScQ

在您的提示下，我参考了里面的prototxt文件，终于可以检测出视频内容了，非常感谢您的意见。
但是检测效果不太理想，漏检率还是很高的，但是检测速度在tx2上有20帧左右，速度还可以。
对比了下直接使用darknet，差距还是很大的。

**Comment by passion3394** [comment]
@dd110hh110 我觉得作者的这种实现方式可能有点问题，期待作者能改进
